clientapi/routing/routing.go

```diff
@@ -263,7 +263,7 @@ func Setup(
 			if !isAllowed {
 				return util.JSONResponse{
 					Code: http.StatusUnauthorized,
-					JSON: jsonerror.Forbidden("Unauthorised"),
+					JSON: jsonerror.Forbidden(""),
```

Comment:
Client gets a 401 with a generic "unknown message". So I had added a message.

Reply:
Can you revert?

Reply:
👍